src/groundstation.py

```diff
@@ -66,7 +66,7 @@ def __zmq__(self, addr):
         libcsp.rtable_load('0/0 ZMQHUB')
 
     def __uart__(self):
-        libcsp.kiss_init('/dev/ttyUSB15', 9600, 512, 'uart')
+        libcsp.kiss_init('/dev/ttyUSB0', 9600, 512, 'uart')
```

Comment:
This is passed as an argument in the updated version, there should be no src/groundstation.py, you should pull from master, perhaps with the `--rebase` option